### DISCUSSION TITLE
No compression_loss needed for QAT?

### DISCUSSION THREAD

**Discussion by EmonLu** [openvinotoolkit/nncf#3024] [Q&A] [answered]
Hi, 

I noticed there're two Usage.md under docs/usage/training_time_compression, one for [QAT](https://github.com/openvinotoolkit/nncf/blob/develop/docs/usage/training_time_compression/quantization_aware_training/Usage.md) and one for [other algorithms](https://github.com/openvinotoolkit/nncf/blob/develop/docs/usage/training_time_compression/other_algorithms/Usage.md). 

1. Why there is no compression_loss in QAT docs?    How does quantization update the parameters like scalefactor and zeropoint? 
2. Is PTQ indispensable in QAT? 
3. Are there any tips for QAT to have a better performance?  e.g. use PTInitialDataloader to initialize

**Comment by alexsu52** [comment, marked as answer]
Hello @EmonLu,

At the first, I would lke to note that `create_compressed_model(...)` API will be deprecated in the next release and `nncf.quantize` is recommended API for [QAT](https://github.com/openvinotoolkit/nncf/blob/develop/docs/usage/training_time_compression/quantization_aware_training/Usage.md). 

> Why there is no compression_loss in QAT docs? How does quantization update the parameters like scalefactor and zeropoint?

QAT uses the model loss that is used for training and does not require additional compression losses.

> Is PTQ indispensable in QAT?

PTQ provides good initialization for QAT to have a better performance. 

> Are there any tips for QAT to have a better performance? e.g. use PTInitialDataloader to initialize

`PTInitialDataloader` will be deprecated in the next release as well. Please take a look at Jupyter notebook on [Quantization Aware Training with NNCF and PyTorch](https://github.com/openvinotoolkit/openvino_notebooks/tree/latest/notebooks/pytorch-quantization-aware-training) which demonstrates how to use NNCF API for QAT and includes general tips to achieve a good performance.
